faf-commons-lobby/src/main/kotlin/com/faforever/commons/lobby/FafLobbyApi.kt

```diff
@@ -104,27 +107,40 @@ interface FafLobbyApi :
 /// * SHARED ENUMS *
 /// ****************
 
-enum class Faction {
-  @JsonProperty("uef")
-  UEF,
+enum class Faction(
+  @JsonValue
+  val string: String,
```

Comment:
maybe `faString` for avoiding a reserved word in Java

Reply:
good point